### PR TITLE
feat: added faqs section

### PR DESCRIPTION
**Pull request by profstrategy**

### CODE REVIEW

File: app/(public-pages)/page.tsx [hunk 1]
```diff
@@ -7,27 +7,30 @@ import StepsToRegister from '@/app/(public-pages)/_components/landing-page/Steps
 import MeetTheTeam from '@/app/(public-pages)/_components/landing-page/MeetTheTeam';
 import WhyUs from '@/app/(public-pages)/_components/landing-page/WhyUs';
 import Benefits from '@/app/(public-pages)/_components/landing-page/Benefits';
+import Faqs from './_components/landing-page/Faqs';
```
Comment (byte-man74) [inline app/(public-pages)/page.tsx:10]: please use absolute imports here

File: app/(public-pages)/_components/landing-page/HeroSection.tsx [hunk 2]
```diff
@@ -127,7 +127,7 @@ export const HeroSection = () => {
             variant="secondary"
             iconPosition="start"
             icon={<FaPlay className="w-3 h-3 sm:w-4 sm:h-4" />}
-            onClick={() => {}}
+            onClick={() =>  router.push('/about')}
```
Comment (byte-man74) [inline app/(public-pages)/_components/landing-page/HeroSection.tsx:130]: use the client-route file and declare this as constant. please dont hard code routes. so incase we have changes one day we wont need to be manually going through every file to change the links

File: app/(public-pages)/_components/landing-page/Faqs.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+                                <AccordionItem value={`item-${idx}`}>
+                                    <AccordionTrigger
+                                        className="flex justify-between w-full"
+                                        onClick={() => handleToggle(idx)}
+                                    >
+                                        <h3 className="text-lg font-semibold text-brand-color xmd:text-start md:text-center">
+                                            {eachitm.question || eachitm.question_1}
+                                        </h3>
+                                        {activeIndex === idx ? <ChevronDown /> : <ChevronUp />}
+                                    </AccordionTrigger>
```
Comment (byte-man74) [inline app/(public-pages)/_components/landing-page/Faqs.tsx:82]: we should map this to prevent repititon.

**Review by byte-man74 — CHANGES_REQUESTED** [review]

File: app/(public-pages)/_components/landing-page/HeroSection.tsx (hunk 2 above)
Comment (profstrategy) [reply app/(public-pages)/_components/landing-page/HeroSection.tsx:130]: Noted boss

File: app/(public-pages)/_components/landing-page/Faqs.tsx (hunk 3 above)
Comment (profstrategy) [reply app/(public-pages)/_components/landing-page/Faqs.tsx:82]: Okay, it was rendered base on the shape of the data. Thanks for the feedback

**Pull request merged**